package.json

```diff
@@ -24,39 +24,37 @@
     "mongodb": "4.3.1",
     "mpath": "0.8.4",
     "mquery": "4.0.2",
-    "ms": "2.1.2",
-    "sift": "13.5.2"
+    "ms": "2.1.3",
+    "sift": "16.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.10.5",
-    "@babel/preset-env": "7.10.4",
-    "@typescript-eslint/eslint-plugin": "5.8.0",
-    "@typescript-eslint/parser": "5.8.0",
+    "@babel/core": "7.17.5",
+    "@babel/preset-env": "7.16.11",
+    "@typescript-eslint/eslint-plugin": "5.12.0",
+    "@typescript-eslint/parser": "5.12.0",
     "acquit": "1.x",
-    "acquit-ignore": "0.1.x",
+    "acquit-ignore": "0.2.x",
     "acquit-require": "0.1.x",
-    "babel-loader": "8.1.0",
+    "babel-loader": "8.2.3",
     "benchmark": "2.1.4",
     "bluebird": "3.7.2",
-    "cheerio": "1.0.0-rc.5",
-    "dox": "0.3.1",
-    "eslint": "8.5.0",
-    "eslint-plugin-mocha-no-only": "1.1.0",
+    "cheerio": "1.0.0-rc.10",
+    "dox": "0.9.0",
```

Comment:
I'd rather upgrade this dep in a separate PR because how we use dox is pretty brittle and we don't have a good way of testing that the dox upgrade didn't break anything other than reviewing all the API docs manually

Reply:
Done 👍